cdc/server.go

```diff
@@ -300,7 +300,8 @@ func (s *Server) setUpDataDir(ctx context.Context) error {
 	for id := range allStatus {
 		info, err := s.etcdClient.GetChangeFeedInfo(ctx, id)
```

Comment:
What about using `GetChangeFeeds` to query all changefeed info. (maybe a little change to this function, unmarshall changefeed info inside)

Reply:
we can add a new function `s.etcdClient.GetAllChangeFeedInfo(ctx)`

Reply:
Fine